src/orquestra/quantum/operators/_pauli_operators.py

```diff
@@ -549,6 +549,12 @@ def __hash__(self):
     def is_constant(self) -> bool:
         return len(self.terms) == 0 or all([term.is_constant for term in self.terms])
 
+    @property
+    def constant_value(self) -> Union[int, float]:
+        return sum(
```

Comment:
1. It shouldn't be sum, there should be only one constant term 🤔  But fine, I see how software can be different from "in principle".
2. I think `Union[int, float]` doesn't really make sense, there is a different type for that, `Number` I think 🤔